typesystem/py.typed

```diff
@@ -0,0 +1 @@
+# Marker file for PEP 561. Typesystem uses inline type hints. See PEP 484 for more information.
```

Comment:
Would probably prefer just a blank `py.typed`, in-line with encode/databases#100 and encode/starlette#455.

Reply:
Fixed